x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -135,7 +146,21 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         const newComment = await find.byCssSelector(
           '[data-test-subj*="comment-create-action"] [data-test-subj="scrollable-markdown"]'
         );
+
         expect(await newComment.getVisibleText()).equal('Test comment from automation');
+        expect(await commentArea.getVisibleText()).equal('');
```

Comment:
Since we have the famous bug of `sometimes comments still remain in markdown after adding it`, do you think it will affect this check randomly?

Reply:
Ah damn it, that is a good point 😬

I removed it from the other place so wanted to have it here but maybe julian's PR fix the timeouts there.